.github/workflows/svelte.yml

```diff
@@ -30,8 +30,11 @@ jobs:
       - name: Install dependencies
         run: npm ci
 
-      #      - name: Run tests
-      #        run: npm test
+      - name: Check Styling
+        run: npm run lint
```

Comment:
Will this only warn, or will it also fail if there is some bad linting?

Reply:
I believe this will fail as well.
Do we want to skip it?

You can see two warnings in this PR (on the changes tab), those are generated by this step.